### PR TITLE
js file, home work: classes

### PR DESCRIPTION
**Pull request by MariiaTyshkovets**
js file, home work: classes

### CODE REVIEW

File: script/script.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+class Circle {
+    constructor (radius) {
+        if (isPositiveNumber(radius)){
+            this.radius = radius;
+        } else {
+            console.log("The radius of circle should be positive number.") ;
+        } 
+    }
+
+    getRadius () {
```
Comment (roskyba) [inline script/script.js]: Try to use `get radius`

File: script/script.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        return this.radius;
+    }
+
+    setRadius (number) {
```
Comment (roskyba) [inline script/script.js]: and `set radius` respectively

File: script/script.js [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        this.color = color;
+        this.inkVolume = 100;
+    }
+
+    inputTextAndPrint() {
+        let text = prompt(`Enter the text.`);
+        let par = document.createElement('p');
+        document.body.appendChild(par);
+        par.style.color = this.color;
+        let str = text.replace(/\s+/g, '');
```
Comment (roskyba) [inline script/script.js:76]: Nicely done with regExp

File: script/script.js [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+            return employee;
+        });
+    }
+
+    getHTML () {
+        let tbody = document.getElementsByClassName("tbody")[0];
+        for (let i = 0; i < employees.length; i++) {
+            let tr = document.createElement("TR");
+            tbody.appendChild(tr);
+            for (const property in employees[i]) {
```
Comment (roskyba) [inline script/script.js:164]: Be careful with `for...in` loop for objects. It will loop through object's prototypes props as well

**Review by roskyba — APPROVED** [review]

File: script/script.js (hunk 2 above)
Comment (MariiaTyshkovets) [reply script/script.js]: змінила вже)

File: script/script.js (hunk 1 above)
Comment (MariiaTyshkovets) [reply script/script.js]: Так і зробила

File: script/script.js (hunk 3 above)
Comment (MariiaTyshkovets) [reply script/script.js:76]: :-)

**Pull request merged**